### PR TITLE
Drop hb logging

### LINKED ISSUE #325: Cron startup error
```

/opt/app/rialto/shared/bundle/ruby/2.5.0/gems/honeybadger-4.1.0/lib/honeybadger/logging.rb:121:in `tty?': closed stream (IOError)
        from /opt/app/rialto/shared/bundle/ruby/2.5.0/gems/honeybadger-4.1.0/lib/honeybadger/logging.rb:121:in `initialize'
        from /opt/app/rialto/shared/bundle/ruby/2.5.0/gems/honeybadger-4.1.0/lib/honeybadger/config.rb:373:in `new'
        from /opt/app/rialto/shared/bundle/ruby/2.5.0/gems/honeybadger-4.1.0/lib/honeybadger/config.rb:373:in `init_logging!'
        from /opt/app/rialto/shared/bundle/ruby/2.5.0/gems/honeybadger-4.1.0/lib/honeybadger/config.rb:146:in `logger'
        from /opt/app/rialto/shared/bundle/ruby/2.5.0/gems/honeybadger-4.1.0/lib/honeybadger/logging.rb:43:in `logger'
        from /opt/app/rialto/shared/bundle/ruby/2.5.0/gems/honeybadger-4.1.0/lib/honeybadger/logging.rb:18:in `debug'
        from /opt/app/rialto/shared/bundle/ruby/2.5.0/gems/honeybadger-4.1.0/lib/honeybadger/worker.rb:56:in `shutdown'
        from /opt/app/rialto/shared/bundle/ruby/2.5.0/gems/honeybadger-4.1.0/lib/honeybadger/agent.rb:275:in `stop'
        from /opt/app/rialto/shared/bundle/ruby/2.5.0/gems/honeybadger-4.1.0/lib/honeybadger/singleton.rb:71:in `block in install_at_exit_callback'
```

### PR DESCRIPTION
**Pull request by aaron-collier**
Fixes #325 

Honeybadger doesn't seem to play well with Yell.

### CODE REVIEW

File: lib/rialto/etl/cli/error_reporter.rb [hunk 1]
```diff
@@ -7,16 +7,12 @@ module Etl
...
       class ErrorReporter
-        extend Logging
 
         # Set up Honeybadger manually, so that we set the logger before honeybadger
         # logs its first message.
         Honeybadger.init!(framework: :ruby, env: ENV['RUBY_ENV'])
         Honeybadger.load_plugins!
         Honeybadger.install_at_exit_callback
-        Honeybadger.configure do |config|
-          config.logger = logger
```
Comment (jcoyne) [inline lib/rialto/etl/cli/error_reporter.rb]: If you do this, will HB log to STDOUT?

Comment (jcoyne) [reply lib/rialto/etl/cli/error_reporter.rb]: If so, does that cause the HB logs to be redirected into the JSON?

Comment (aaron-collier) [reply lib/rialto/etl/cli/error_reporter.rb]: I think I need to add a honeybadger.log file to this, I did that locally and it logged there fine.

**Comment by mjgiarlo** [comment]
Ha, or yeah, have HB log to its own file. :)

**Pull request merged**